docs/tutorial.rst

```diff
@@ -618,6 +618,102 @@ Once you are satisfied that the demographic history that you have built
 is correct, it can then be simulated by calling the :func:`.simulate`
 function.
 
+-------------
+Census events
+-------------
+
+Census events allow you to add a node to each branch of the tree sequence at a given time during the simulation. This can be useful when you wish to study haplotypes that are ancestral to your simulated sample.
```

Comment:
These lines are very long --- can you break them up a bit? This makes it easier to do diffs and stuff later.

Reply:
Good to note that this allows to figure out *where* all the lineages are at this time as well, as this is the main application I think.